tasks/main-clevis.yml

```diff
@@ -4,6 +4,18 @@
     name: "{{ __nbde_client_packages }}"
     state: present
 
+- name: Enable clevis askpass unit
+  service:
+    name: clevis-luks-askpass.path
+    enabled: yes
+
+- name: Generate nbde_client dracut config
+  template:
+    src: nbde_client.conf
+    dest: /etc/dracut.conf.d/nbde_client.conf
+    owner: root
+    mode: u=rw
```

Comment:
Are write permissions required for this file? I guess 0444 permission should be enough

Reply:
Done.